config/webpack.dev.config.js

```diff
@@ -19,6 +19,38 @@ if (result.error) {
   throw result.error;
 }
 
+function getLocalAliases() {
```

Comment:
This function is taken from `frontend-build`.

Reply:
interesting.. perhaps we can find a way to use what's in frontend-build later?

Reply:
Makes sense to duplicate here for now. Once we're fully transitioned this repo `frontend-build` in the future we'll be able to remove this duplicate function.